pegasus/sites.v3/code.org/styles_min/040-page.css

```diff
@@ -527,6 +527,7 @@ nav.main .user_menu {
   margin: 0;
   display: flex;
   gap: 2px;
+  cursor: pointer;
```

Comment:
This adds a pointer on the code.org top nav buttons — forgot to update this in the code.org top nav refactor PR!